packages/react-examples/src/react/ScrollablePane/ScrollablePane.doc.tsx

```diff
@@ -19,7 +19,7 @@ export const ScrollablePanePageProps: IDocPageProps = {
       isScrollable: false,
     },
     {
-      title: 'DetailsList Locked Header',
+      title: 'DetailsList Fixed Header without ScrollablePane',
```

Comment:
I wonder if rather than having the example here, it ought to be moved under DetailsList (and you can link to it from the markdown below). Unfortunately this requires creating a few boilerplate files, so if you want I can make that change and push it to the PR.